.gitignore

```diff
@@ -20,8 +20,7 @@ priv/server.key
 priv/server.pem
 
 # generated static files
-priv/static/js/
-priv/static/css/
+priv/static/
```

Comment:
Do we need to retain https://github.com/mozilla/reticulum/blob/master/priv/static/favicon-spoke.ico and https://github.com/mozilla/reticulum/blob/master/priv/static/spoke-preview.png ?

Reply:
From what I can tell these are all generated files so I don't think so, but leaving them in place just in case.